site/src/Librecores/ProjectRepoBundle/Controller/DefaultController.php

```diff
@@ -94,96 +96,17 @@ public function userOrgSettingsAction(Request $request, $userOrOrganization)
-            'label' => 'Organizations',
-        ));
-        $searchQueryForm->handleRequest($req);
-
-        // Form validation: If we encounter any invalid value, simply
-        // redirect to an empty search form.
-        if (!empty($searchQuery->getQ()) && !$searchQueryForm->isValid()) {
-            return $this->redirectToRoute($req->get('_route'));
-        }
-
-        // Handle switching of search type
-        // In order to have always copy&paste-able URLs, we adjust the type
-        // based on the button click event and then redirect to a "nice" URL
-        // with the search results.
-        $redirect = false;
-        if ($searchQueryForm->get('search_projects')->isClicked()) {
-            $searchQuery->setType(SearchQuery::TYPE_PROJECTS);
-            $redirect = true;
-        }
-        if ($searchQueryForm->get('search_users')->isClicked()) {
-            $searchQuery->setType(SearchQuery::TYPE_USERS);
-            $redirect = true;
-        }
-        if ($searchQueryForm->get('search_orgs')->isClicked()) {
-            $searchQuery->setType(SearchQuery::TYPE_ORGS);
-            $redirect = true;
-        }
-        if ($redirect) {
-            return $this->redirectToRoute(
-                $req->get('_route'),
-                [ 'q' => $searchQuery->getQ(), 'type' => $searchQuery->getType()]
-            );
-        }
-
-        // No query string given: no search necessary
-        if (empty($searchQuery->getQ())) {
-            return $this->render(
-                'LibrecoresProjectRepoBundle:Default:project_search.html.twig',
-                [
-                    'search_query_form' => $searchQueryForm->createView(),
-                    'search_query' => $searchQuery,
-                    'projects' => [],
-                    'users' => [],
-                    'orgs' => [],
-                ]
-            );
-        }
-
-        // Get the results
-        $projects = array();
-        $users = array();
-        $orgs = array();
-
-        // Search for projects
-        if ($searchQuery->getType() === SearchQuery::TYPE_PROJECTS) {
-            $projects = $this->getDoctrine()
-                ->getRepository('LibrecoresProjectRepoBundle:Project')
-                ->findByFqnameFragment($searchQuery->getQ());
-        }
-
-        // Search for users
-        if ($searchQuery->getType() === SearchQuery::TYPE_USERS) {
-            $userManager = $this->get('fos_user.user_manager');
-            $users = $userManager->findUsersBySearchString($searchQuery->getQ());
-        }
+        $searchType = $req->get('type');
+        $searchQuery = $req->get('query');
 
-        // Search for organizations
-        if ($searchQuery->getType() === SearchQuery::TYPE_ORGS) {
-            $orgs = $this->getDoctrine()
-                ->getRepository('LibrecoresProjectRepoBundle:Organization')
-                ->findByFragment($searchQuery->getQ());
-        }
+        // If searchType is null
```

Comment:
... then what? How about "search projects by default" (or remove the comment altogether)